src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -891,12 +892,12 @@ private static function createAttachmentList($item, $type)
 	private static function mentionCallback($match)
 	{
 		if (empty($match[1])) {
-			return;
+			return '';
 		}
 
 		$data = Contact::getDetailsByURL($match[1]);
-		if (empty($data) || empty($data['nick'])) {
```

Comment:
Why do you remove the check? We shouldn't trust implicit returns, I think.

Reply:
See line 899.

Reply:
I mean the check if `$data` isn't null.

Reply:
It's included in `empty($data['nick'])`.